app/controllers/concerns/api/articles_controller.rb

```diff
@@ -109,7 +109,7 @@ def unpublish
       authorize @article, :revoke_publication?
 
       if Articles::Unpublish.call(@user, @article)
-        render head: :ok
```

Comment:
Based on my research it's more customary to use HTTP code 204 NO CONTENT if there's no information being returned from the server (as is happening with the unpublished endpoint): is that ok with you @fdocr?